tool/rig/main.go

```diff
@@ -197,6 +197,11 @@ func freeze(ctx context.Context, client *kubernetes.Clientset, config *rest.Conf
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	committed, _ := cs.IsCommitted(ctx, namespace, changeset.Name)
```

Comment:
Do we have to ignore the error here and in all other instances?

Reply:
I believe it's safe as iff `!committed`, then we fall through to the next resource operation which will likely fail with the same error.